File: BAHASAC_1/PRAK110-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>
#include <math.h>
int main()
{
    int A = 12;
    int C = 5;
    int B = sqrt (A*A + C*C);
    int K = A + B + C;
    int L = 0.5*C*A;

    printf("Diketahui :\n");
    printf("Alas = %.d cm\n", C);
    printf("Tinggi = %d cm\n\n", A);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", A);
    printf("Sisi B = %d cm\n", B);
    printf("Sisi C = %d cm\n", C);
    printf("Keliling = %d cm\n", K);
    printf("Luas = %d cm\n", L);
    
    return 0;
}

File: PHYTON_1/PRAK110-2310817310008-MUHAMMADRIZKIRAMADHAN.py
import math

A = 12
C = 5
B = round(math.sqrt(A*A + C*C))
K = round(A + B + C)
L = round(0.5*C*A)

print("Diketahui :")
print(f"Alas = {C} cm")
print(f"Tinggi = {A} cm\n\n")
print("Jawab :")
print(f"Sisi A = {A} cm")
print(f"Sisi B = {B} cm")
print(f"Sisi C = {C} cm")
print(f"Keliling = {K} cm")
print(f"Luas = {L} cm")